crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -42,24 +48,53 @@ pub struct StandardMaterial {
     /// it just adds a value to the color seen on screen.
     pub emissive: Color,
 
+    /// The [`emissive`] as a texture.
```

Comment:
Same here; this modulates the `emissive` color, this doesn't replace it.

Reply:
The entirety of the rest of this doc comment explains how it mixes with the `emissive` value. I can't really make it clearer than that. I'm not sure what you are suggesting here.

Reply:
My problem is the `as` which makes it sound like it's overwriting the scalar value. I agree to point to another part of the docs if we do so explicitly with a sentence, and do not contradict ourselves. For me, there's a contradiction in the terminology here.